hummingbot/connector/exchange/ascend_ex/ascend_ex_order_book_tracker.py

```diff
@@ -89,17 +88,14 @@ async def _track_single_book(self, trading_pair: str):
                         diff_messages_accepted = 0
                     last_message_timestamp = now
                 elif message.type is OrderBookMessageType.SNAPSHOT:
-                    past_diffs: List[AscendExOrderBookMessage] = list(past_diffs_window)
-                    # only replay diffs later than snapshot, first update active order with snapshot then replay diffs
+                    past_diffs = list(past_diffs_window)
                     replay_position = bisect.bisect_right(past_diffs, message)
                     replay_diffs = past_diffs[replay_position:]
-                    s_bids, s_asks = active_order_tracker.convert_snapshot_message_to_order_book_row(message)
-                    order_book.apply_snapshot(s_bids, s_asks, message.update_id)
-                    for diff_message in replay_diffs:
-                        d_bids, d_asks = active_order_tracker.convert_diff_message_to_order_book_row(diff_message)
-                        order_book.apply_diffs(d_bids, d_asks, diff_message.update_id)
-
+                    order_book.apply_snapshot(message.bids, message.asks, message.update_id)
```

Comment:
Similar for `apply_snapshot()` it expects bids and asks to be `List[OrderBookRow]`